hw3/01-characters.js

```diff
@@ -0,0 +1,43 @@
+// url for the Thrones API
+const url = 'https://thronesapi.com/api/v2/Characters';
+
+fetch(url)
+.then(response => response.json())
+.then(characters => {
+    const characterList = document.getElementById('characters');
+    
+    characters.forEach(character => {
+      const { firstName, lastName, imageUrl, title } = character;
+
+      const characterContainer = document.createElement('section');
+      characterContainer.classList.add('character-container');
+
+      const characterImage = document.createElement('img');
+      characterImage.src = imageUrl;
+      characterImage.alt = imageUrl;
+
+      const characterName = document.createElement('h2');
+      characterName.textContent = `${firstName} ${lastName}`;
+
+      const characterTitle = document.createElement('h3');
+      characterTitle.textContent = title;
+
+      characterImage.style = "margin: 25px; width: 200px; height: 220px"
+      characterName.style = "margin-bottom: 20px; text-align: center; font-size: 20px; font-weight: bolder; line-height: 20px;"
+      characterTitle.style = "margin-left: 22px; text-align: center; margin-bottom: 20px; font-size: 18px; font-weight: bolder; line-height: 20px; width: 200px"
+
+      characterContainer.appendChild(characterImage);
+      characterContainer.appendChild(characterName);
+      characterContainer.appendChild(characterTitle);
```

Comment:
Using the regular append() method all three child elements (characterImage, characterName, and characterTitle) could be appended to characterElement at the same time. This is not necessary, but it would save a few lines while still being readable.

The append() method works like appendChild(), but lets you pass in multiple items.

```suggestion
      characterContainer.append(characterImage, characterName, characterTitle );
```

Reply:
Thank you for the suggestion, @JoshuaWachana. You're absolutely right! Instead of appending the child elements one by one, I can use the append() method to append all three child elements (characterImage, characterName, and characterTitle) to characterContainer at the same time, which would make the code more concise while maintaining readability